client/blocks/post-share/index.jsx

```diff
@@ -141,6 +141,13 @@ class PostShare extends Component {
 
 	toggleSharingPreview = () => {
 		const showSharingPreview = ! this.state.showSharingPreview;
+
+		if ( showSharingPreview ) {
+			document.documentElement.classList.add( 'no-scroll', 'is-previewing' );
```

Comment:
Yeah looked weird to me too, I found this in the web preview: https://github.com/Automattic/wp-calypso/blob/master/client/components/web-preview/index.jsx#L44